app/services/flow/flows/provider_start.rb

```diff
@@ -61,18 +61,28 @@ class ProviderStart < FlowSteps # rubocop:disable Metrics/ClassLength
         },
         check_provider_answers: {
           path: ->(application) { urls.providers_legal_aid_application_check_provider_answers_path(application) },
-          forward: :check_benefits
+          forward: ->(application) do
+            if Setting.override_dwp_results?
+              application.non_passported? ? :confirm_dwp_non_passported_applications : :check_benefits
+            else
+              :check_benefits
+            end
+          end
         },
         check_benefits: {
           path: ->(application) { urls.providers_legal_aid_application_check_benefits_path(application) },
-          forward: ->(application, dwp_override_non_passported) do
```

Comment:
Is this dwp_override supposed to be removed?

Reply:
Good catch. This ties in with my comments on other files. Obviously I didn't spot all of them 🤦